app/commands/v2/put_content.rb

```diff
@@ -87,9 +87,10 @@ def check_update_type
       end
 
       def create_links(edition)
-        return if payload[:links].nil?
+        links = (payload[:links] || {}).merge(fetch_embedded_content(edition))
```

Comment:
We need to think about when embedded editions would be removed - Bruce says it's a bit weird in that you need to provide a key with an empty value to clear out an existing link (rather than just not providing it).